Storybook/Pages/GameView/AnimationPage/AnimationPageScene.swift

```diff
@@ -11,60 +11,104 @@ import UIKit
-        //        text.outlinedText = "Ada yang berbentuk \(textAnimateOne), \(textAnimateTwo) dan \(textAnimateThree) "
+        let rawLabels = story?.labels as? [String]
         
-        text.fontColor = UIColor.white
-        text.position = CGPoint(x: 0, y: 175)
-        text.zPosition = 15
-        text.preferredMaxLayoutWidth = 700
-        text.numberOfLines = 2
-        //        text.attributedText =
+        if let labels = rawLabels {
+            for (idx, label) in labels.enumerated() {
+                let textScene = SKLabelNode(fontNamed: "Poppins-Black")
+                textScene.text = label
+                textScene.fontSize = 50
+                textScene.fontColor = SKColor.white
+                textScene.position = CGPoint(x: 0, y: Int(frame.height)/3-idx*60)
+                textScene.zPosition = 100
+                textScene.addStroke(color: textBorder, width: 7.0)
+                addChild(textScene)
+            }
+        }
         
-        //        let attributes: [NSAttributedString.Key: Any] = [
-        //            .strokeWidth: 5.0,
-        //            .strokeColor: UIColor.black
-        //        ]
+        box.position = CGPoint(x: -265, y: -245)
+        box.zPosition = 15
+        box.size = CGSize(width: 178, height: 191)
         
-        //        let attributedString = NSAttributedString(string: text, attributes: attributes)
+        cone.position = CGPoint(x: -25, y: -250)
+        cone.zPosition = 15
+        cone.size = CGSize(width: 208, height: 194)
         
-        //        text.horizontalAlignmentMode = .center
+        ball.position = CGPoint(x: 204, y: -235)
+        ball.zPosition = 15
+        ball.size = CGSize(width: 198, height: 217)
         
-        addChild(text)
+        addChild(box)
+        addChild(cone)
+        addChild(ball)
     }
     
     override func didMove(to view: SKView) {
+        do {
+            let fetchRequest = Stories.fetchRequest()
+            let orderPredicate = NSPredicate(format: "order == 0")
+            let challengeNamePredicate = NSPredicate(format: "challengeName == %@", (challengeName ?? "") + "_animate")
+            fetchRequest.predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [
+                challengeNamePredicate, orderPredicate
+            ])
+            fetchRequest.fetchLimit = 1
+            story = try context.fetch(fetchRequest)[0]
+            
+            fetchRequest.predicate = NSPredicate(format: "challengeName == %@", (challengeName ?? "") + "_animate")
+
+            totalStories = try context.count(for: fetchRequest)
+        } catch let error as NSError {
+            print(error)
+            print("error while fetching data in core data!")
+        }
+        
         self.setupPlayer()
     }
     
     override func getNextScene() -> SKScene? {
-        let scene = SKScene(fileNamed: "ShapeGameScene") as! ShapeGameScene
+//        if Int(idxScenePreAnimate) < totalStories ?? 0 {
```

Comment:
Please remove this if you don't need it anymore, let's get used to removing unused code. @adamfiadi7007